conf/imperial.config

```diff
@@ -1,37 +1,92 @@
 //Profile config names for nf-core/configs
 
-params {
-  // Config Params
-  config_profile_description = 'Imperial College London - HPC Profile -- provided by nf-core/configs.'
-  config_profile_contact = 'Combiz Khozoie (c.khozoie@imperial.ac.uk)'
-  config_profile_url = 'https://www.imperial.ac.uk/admin-services/ict/self-service/research-support/rcs/'
-
-  // Resources
-  max_memory = 256.GB
-  max_cpus = 32
-  max_time = 72.h
-}
+profiles {
+    imperial {
+        params {
+            // Config Params
+            config_profile_description = 'Imperial College London - HPC Profile -- provided by nf-core/configs.'
+            config_profile_contact = 'Combiz Khozoie (c.khozoie@imperial.ac.uk)'
+            config_profile_url = 'https://www.imperial.ac.uk/admin-services/ict/self-service/research-support/rcs/'
```

Comment:
I'd just keep all that outside any profile, since it's your default, and add whatever you want from the medbio in the profile scope.
cf this is what we've done with the uppmax and the included dev profile:
https://github.com/nf-core/configs/blob/master/conf/uppmax.config

Reply:
Fixed in https://github.com/nf-core/configs/pull/321/commits/3ce32f50c45b44ca1226f1a60b6cf821a26315df